src/firmware/app/world/firmware_robot_test.cpp

```diff
@@ -158,10 +155,9 @@ TEST_F(FirmwareRobotTest, getRobotConstants)
 {
     RobotConstants_t constants = app_firmware_robot_getRobotConstants(firmware_robot);
 
-    EXPECT_NEAR(1.1, constants.mass, 1e-5);
-    EXPECT_NEAR(1.2, constants.moment_of_inertia, 1e-5);
-    EXPECT_NEAR(1.3, constants.robot_radius, 1e-5);
-    EXPECT_NEAR(1.4, constants.jerk_limit, 1e-5);
+    EXPECT_NEAR(1.1, constants.mass_kg, 1e-5);
+    EXPECT_NEAR(1.2, constants.moment_of_inertia_kg_m_2, 1e-5);
+    EXPECT_NEAR(1.4, constants.jerk_limit_kg_m_per_s_3, 1e-5);
```

Comment:
Just confirming if you purposely omitted the robot_radius test here?

Reply:
not on purpose, fixed